김윤건/week1./주차 요금 계산.js

```diff
@@ -0,0 +1,53 @@
+function solution(fees, records) {
+  const answer = [];
+  const [defaultTime, defaultFee, unitTime, unitFee] = fees;
+  const timeStore = {};
+  
+  records.forEach(record => {
+      const [time, carNumber, state] = record.split(' ');
+      const [hour, min] = time.split(':');
+      const convertedTime = +hour * 60 + +min;
```

Comment:
보다 명시적인 숫자 변환을 위해  Number함수 사용을 추천드립니다.
```suggestion
      const convertedTime = Number(hour) * 60 + Number(min);
```

Reply:
확실히 Number가 더 명시적인 것 같습니다.
습관적으로 사용하기도 하고 이 정도는 이해하고 넘어갈 수 있는 부분이라는 생각에 적었던 것 같습니다.
감사합니다~!